src/actors/process_list/process_list.py

```diff
@@ -0,0 +1,74 @@
+import os
+import sys
+import json
+
+
+def readlink(path, filename):
+    return os.readlink(os.path.join(path, filename))
+
+
+def get_environ(pid):
+    try:
+        with open(os.path.join(pid, 'environ'), 'rb') as f:
+            output = {}
+            environ = f.read().split("\x00")
+            for env in environ:
+                if env:
+                    name, val = env.split('=', 1)
+                    output[name] = val
+            return output
+    except EnvironmentError as exc:
+        sys.stderr.write(str(exc) + "\n")
```

Comment:
1. instead of using sys.stderr.write use print with redirect to stderr
2. add ```return None``` statement following the rule "Explicit is better than implicit."

Everywhere :)